src/builder/openChrome.applescript

```diff
@@ -0,0 +1,49 @@
+(*
+Copyright (c) 2015-present, Facebook, Inc.
+All rights reserved.
+
+This source code is licensed under the BSD-style license found in the
+-- LICENSE file in the root directory of this source tree. An additional grant
+of patent rights can be found in the PATENTS file in the same directory.
+*)
```

Comment:
I don't want to be annoying, but I just don't want a explicit copyright in a file, that is already a copy of another (copyrighted, as there is not explicit license ?) script (https://gist.github.com/mayoff/1138816, 5yo gist).
Btw, it seems there is an improved version in a comment https://gist.github.com/mayoff/1138816#gistcomment-1821795 (not sure what the real diff)